### DISCUSSION TITLE
authentik default login method

### DISCUSSION THREAD

**Discussion by VPaulV** [goauthentik/authentik#5470] [Q&A]
Hello everyone,

I was wondering if there is a way to establish a default login method. I have configured OAuth2 login using Mailcow, and when I access an application that is secured by Authentik, I would like to be immediately redirected to Mailcow's OAuth without being prompted for the default login. Is there a way to achieve this?

Thank you!

**Comment by VPaulV** [comment]
Hi Guys, is there any chance somebody could provide an update on this?

**Comment by VPaulV** [comment]
What I would like to achieve is, instead of seeing this on login:
![image](https://github.com/goauthentik/authentik/assets/5863225/376892a9-9d44-485a-8f07-8bc63fbe15bc)
I want to be redirected to Mailcow OAuth2 login immediately

**Comment by l4rm4nd** [comment]
Hi @VPaulV,

I've done exactly this for Azure AD. May read my blog post [here](https://blog.lrvt.de/authentik-traefik-azure-ad/#disable-local-auth) about it.

You have to adjust the default authentication flow and remove all user fields.

To do so:

1. Log into Authentik as admin and visit the `admin interface`
2. Browse to `Flows and Stages > Flows`
3. Select `default-authentication-flow`
4. Select `Stage-Bindings` at the top
5. Select `default-authentication-identification` and `Edit`
6. Drop down `Sources` and select your Mailcow source only.
7. At `Stage-specific settings`, unselect the fields `Username` and `Email` at the `User fields` section. Use SHIFT and/or CRTL to unselect all entries.
8. Finish the flow setup by clicking `Update`

Now try again. No Authentik login screen anymore.

![image](https://github.com/goauthentik/authentik/assets/21357789/2d5123e8-6136-4a3b-a63a-80fe594ccc7a)

**Reply by VPaulV** [reply]
Thank you! That is exactly what I was looking for
